### PR TITLE
Adds steam transport machine

### PR DESCRIPTION
**Pull request by Coxswain-Navigator**
## About The Pull Request
Adds steam transport machine, a new HE-level abnormality. It serves as an excellent early-game option that becomes more dangerous as the round progresses, becoming stronger and harder to keep contained.
In terms of behavior, it simply roams around blasting people in range with steam or hitting them with its arms. One notable detail is its tendency to pause while firing.

## Why It's Good For The Game
New abnormality with an interesting gimmick

## Changelog
:cl:
add: Added steam transport machine
/:cl:

### CODE REVIEW

**Comment by JackFrost2424** [comment]
Finally. Agents can spend 50 hours fighting against Steam Transport in LC13.

File: code/modules/mob/living/simple_animal/abnormality/he/steam_transport_machine.dm [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+	icon_living = "steam"
+	maxHealth = 1600
+	health = 1600
+	ranged = TRUE
+	attack_sound = 'sound/effects/hit_kick.ogg'
+	friendly_verb_continuous = "bonks"
+	friendly_verb_simple = "bonk"
+	attack_verb_continuous = "smashes"
+	attack_verb_simple = "smash"
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.5, WHITE_DAMAGE = 1, BLACK_DAMAGE = 2, PALE_DAMAGE = 1.5)
```
Comment (TheBronJameOffical) [inline code/modules/mob/living/simple_animal/abnormality/he/steam_transport_machine.dm:15]: Brute doesn't have to be defined in this list anymore.

Comment (Coxswain-Navigator) [reply code/modules/mob/living/simple_animal/abnormality/he/steam_transport_machine.dm:15]: A code improvement to make it standard seems like a good idea

File: code/modules/paperwork/records/info/he.dm [hunk 2]
```diff
@@ -480,5 +480,13 @@
...
+//Steam Transport Machine
+/obj/item/paper/fluff/info/he/steam
+	abno_type = /mob/living/simple_animal/hostile/abnormality/steam
+	abno_code = "O-05-197"
+	abno_info = list(
+		"When the work result was neutral, the Qliphoth counter lowered at a moderate probability.",
+		"When the work result was bad, the Qliphoth counter lowered.",
+		"When a certain amount of time passed, the abnormality \"activated\". Each activation increased the severity of the activation effects.",
+		"When activated, the abnormality's maximum Qliphoth counter lowered, all work types became less effective, work damage increased, and the abnormality generally became more aggressive.",
+		"At 1 hour and 15 minutes the maximum Qliphoth counter reduced to 1 and the abnormality became extremely dangerous.")
```
Comment (LanceSmites328) [inline code/modules/paperwork/records/info/he.dm:492]: Maximum Qliphoth does not lower. Only current post-work. It also notably does not update this while breaching.

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]
Why take this? It has normal work rates that only go down with time and eventually scales work damage to be above NT's 16. So it just becomes an unworkable monster later with it's best rate being 36%, why bother?

File: code/modules/paperwork/records/info/he.dm (hunk 2 above)
Comment (Coxswain-Navigator) [reply code/modules/paperwork/records/info/he.dm:492]: Entirely forgot to do this

**Review by LanceSmites328 — APPROVED** [review]

**Pull request merged**